### DISCUSSION TITLE
Congratulations 🎉

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#176] [General]
The first official release of the plugin is awesome.
I can't believe how far the plugin has come!
This is a big moment. Now we can only expect more awesome features to be added in the near future

Thank you @GoodestEnglish and all the other contributors for making this practice plugin.
This plugin beats even the costliest practice plugins and it's free.

**Comment by GoodestEnglish** [comment]
Thank you. Appreciate your support. I believe you are the one of the user who followed Eden from the first few beta releases. So that's really cool!

If you are not come from built by bit, let me tell you some back story, Eden was a plugin which I used in my old server (with multi-server support, but removed). Because of I am now in university, I don't have time to maintain such a huge server, I decided to recode part of Eden and open source it.

Here's a screenshot of my old server, running Eden with 100 players
![image](https://user-images.githubusercontent.com/45838505/225199483-1e40b9a0-9a1f-4d94-95ba-22ac082b6b31.png)

**Comment by GoodestEnglish** [comment]
For more context, the next part I am working on in Eden is SumoEvent. Should be having a beta version in the coming days.

**Reply by HierarchyMan** [reply]
That's really amazing
I'm glad you remembered me.

**Comment by HappyAreaBean** [comment]
cool
